config.md

```diff
@@ -265,6 +265,27 @@ Annotations are key-value maps.
 }
 ```
 
+## HypervisorPath
```

Comment:
@WeiZhang555 - yes, making ``--runtime`` a container-specific flag would indeed solve the issue. +1.

The only complication would be that the user would need to remember to specify it to "``docker run``". Maybe it would be useful to be able to configure ``dockerd`` to specify a default runtime to use? That way, if the user runs, "``docker run $image``", they get the admin-configured default runtime **unless** they explicitly specify "``docker run --runtime=$foo $image``"?